project.mak

```diff
@@ -71,19 +70,19 @@ tests:
 	make -C tests test
 
 zarith.cma: $(MLSRC:%.ml=%.cmo)
-	$(OCAMLMKLIB) $(DEBUG) -failsafe -o zarith $+ $(LIBS)
+	$(OCAMLMKLIB) -failsafe -o zarith $+ $(LIBS)
```

Comment:
Why is debug gone ?

Reply:
Arf, I'm wrong about the destination of this PR (GitHub redirect me directly to this repository but the PR is addressed for mirage/Zarith). Again, sorry for the noise.

Then, specially about your comment, it helps us to cross compile Zarith for some specific targets.